app/containers/TwoFactor/index.tsx

```diff
@@ -55,15 +57,31 @@ const methods: IMethods = {
 	}
 };
 
-const TwoFactor = React.memo(({ isMasterDetail }: { isMasterDetail: boolean }) => {
-	const { theme } = useTheme();
+const TwoFactor = React.memo(() => {
```

Comment:
I thought we would replace the text with a proper button, since the user wasn't really understanding that text was a touchable. Can you do it?